the solution for 0x11-python-network_1
